back-end/resources/manuals/room_manual.md

```diff
@@ -131,7 +131,7 @@ This can be displayed in the front-end, so should be readable and in Dutch.
                 - `operator`: this can `AND` or `OR`
                 - `list`: this is an array of constraints / logical operators
             2. Constraint
-                - `comparison`: this is the type of comparison and can be `eq`, `lt`, `gt`, `contains` , `lte`, `gte`, `not`. However, only `eq` will work on all types, `lt`, `gt`, `lte`, `gte` only on numeric, and `contains` only on arrays, `not` does not work on booleans.
+                - `comparison`: this is the type of comparison and can be `eq`, `lt`, `gt`, `contains` , `lte`, `gte`, `not`, `eqwt` and `not eqwt`. However, only `eq` will work on all types, `lt`, `gt`, `lte`, `gte` only on numeric, and `contains` only on arrays, `not` does not work on booleans. `eqwt` and `not eqwt` will only work on strings, they work like `eq` and `not`, they do however allow for typos, and they ignore capital letters.
```

Comment:
can you add explanation of the name `with typo`?

Reply:
done